app/Main.hs

```diff
@@ -536,6 +541,14 @@ addFromBower name = do
   where
     stripBowerNamePrefix s = fromMaybe s $ T.stripPrefix "purescript-" s
     mkPackageName' = Bifunctor.first show . mkPackageName . stripBowerNamePrefix
+    unpackSpecifier = second (fmap T.tail . textToMaybe) . T.breakOn "#"
+
+maybeE :: b -> Maybe a -> Either b a
+maybeE b = maybe (Left b) Right
```

Comment:
called `note` usually, this should be available in our dependencies

Reply:
`note` (most commonly) comes from the `Control.Error.Util` module:
https://www.stackage.org/haddock/lts-11.7/errors-2.2.5/Control-Error-Util.html#v:note

A quick glance at the reverse dependency search [here](https://packdeps.haskellers.com/reverse/errors) shows that none of [psc-package's dependencies](https://github.com/purescript/psc-package/blob/master/psc-package.cabal#L17-L28) pull it in.

Stack says that adding it in also pulls these two dependencies in transitively:
```
safe-0.3.17
unexceptionalio-0.3.0
```